### PR TITLE
add gauss-otsu-labeling

### PR DESCRIPTION
**Pull request by haesleinhuepf**

### CODE REVIEW

**Comment by StRigaud** [comment]
I will fix the format when the rest of the CI is passing

**Comment by haesleinhuepf** [comment]
Locally the test passes. Let's see...

File: clic/include/tier6.hpp [hunk 1]
```diff
@@ -51,6 +51,33 @@ erode_labels_func(const Device::Pointer & device,
...
+ *  The outline_sigma parameter allows tuning the segmentation result. Under the hood,
+ *  this filter applies a Gaussian blur, Otsu-thresholding [1] and connected component labeling [2]. The
+ *  thresholded binary image is flooded using the Voronoi tesselation approach starting from the found local maxima.
+ *
+ * @param device Device to perform the operation on. [const Device::Pointer &]
+ * @param src0 intensity image to add labels [const Array::Pointer &]
+ * @param dst Output label image. [Array::Pointer ( = None )]
+ * @param outline_sigma Gaussian blur sigma along all axes [float ( = 0 )]
+ * @return Array::Pointer
+ * @link https://ieeexplore.ieee.org/document/4310076
```
Comment (StRigaud) [inline clic/include/tier6.hpp:70]:
```suggestion

 * @link https://ieeexplore.ieee.org/document/4310076
```

**Comment by StRigaud** [comment]
all good, I will fix the format quickly :)

**Pull request merged**